libcxx/test/support/invocable_with_telemetry.h

```diff
@@ -0,0 +1,100 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef TEST_SUPPORT_INVOCABLE_WITH_TELEMETRY_H
+#define TEST_SUPPORT_INVOCABLE_WITH_TELEMETRY_H
+
+#include <cassert>
+#include <concepts>
+#include <utility>
+
+#if TEST_STD_VER < 20
+#  error invocable_with_telemetry requires C++20
+#else
+template <class F>
+class invocable_with_telemetry {
+public:
+  invocable_with_telemetry() = default;
```

Comment:
A default constructed instance of this type cannot be copied, moved, or invoked. Was this intentional?
Given the type design that requires external memory be provided, I think we should delete the default constructor.

Reply:
I'm going to delete this for now.